components/skeleton.tsx

```diff
@@ -46,3 +46,31 @@ export const CommentSkeleton = () => {
     </div>
   );
 };
+
+// FIXME: 왜 w값이 제대로 잡히지 않는가?! (33rem이 잡히지 않고 더 적게 잡힘..)
+/** 기술블로그 메인 - 스켈레톤 */
+export const TechMainSkeleton = () => {
+  return (
+    <div className='flex flex-row  py-[1.6rem] gap-[3.6rem]'>
+      <div className='w-[33rem] h-[18.4rem] rounded-[1.6rem] bg-[#29292E] relative skeleton-item overflow-hidden'></div>
+      <div className='w-[100%] flex flex-col gap-[1.2rem] justify-center'>
```

Comment:
```suggestion
    <div className='flex flex-row py-[1.6rem] gap-[3.6rem]'>
      <div className='w-[24rem] h-[18.4rem] rounded-[1.6rem] bg-[#29292E] relative skeleton-item overflow-hidden'></div>
      <div className='flex flex-col flex-1 gap-[1.2rem] justify-center'>
```

이렇게 수정하면 될 것 같아용! 기존에 w-full 로 하게되면 flex 컨테이너의 너비를 채우려고 하기 때문에 `flex-1` 로 수정했어요!

Reply:
오! 맞아요 저 100%를 지우면 잘 잡히는데, 나머지를 full로 어떻게 채워야하나 했는데,
flex-1을 사용하면 되군요!!! 감사합니당❤️